packages/components/app/styles/components/form/checkbox.scss

```diff
@@ -80,6 +80,22 @@ $hds-form-checkbox-control-border-radius: 3px;
     border-color: var(--token-color-palette-red-400);
   }
 
+  // TODO add handling of focus-visible
+  &.hds-form-checkbox--is-invalid:focus:not(:checked),
+  &.hds-form-checkbox--is-invalid.mock-focus:not(:checked),
+  &.hds-form-checkbox--is-invalid:focus:checked,
+  &.hds-form-checkbox--is-invalid.mock-focus:checked {
+    border-color: var(--token-color-focus-critical-internal);
+    outline-color: var(--token-color-focus-critical-external);
+  }
+  &.hds-form-checkbox--is-invalid:active:not(:checked),
+  &.hds-form-checkbox--is-invalid.mock-active:not(:checked),
+  &.hds-form-checkbox--is-invalid:active:checked,
+  &.hds-form-checkbox--is-invalid.mock-active:checked {
+    background-color: var(--token-color-palette-red-400);
+    border-color: var(--token-color-palette-red-400);
+  }
```

Comment:
We shouldn't need the active ones anymore, right?

Reply:
You're right! 🤦‍♂️ (I fixed the conflicts in the wrong way).